spec/view_objects/result_filters/new_filters_view_spec.rb

```diff
@@ -0,0 +1,260 @@
+        let(:params) { { qualifications: %w[Other PgdePgceWithQts] } }
+
+        it { is_expected.to eq(false) }
+      end
+
+      context 'when QtsOnly param is present' do
+        let(:params) { { qualifications: %w[QtsOnly PgdePgceWithQts] } }
+
+        it { is_expected.to eq(true) }
+      end
+
+      context 'when qualifications is empty' do
+        let(:params) { { qualifications: [] } }
+
+        it { is_expected.to eq(false) }
+      end
+
+      context 'when parameters are empty' do
+        let(:params) { {} }
+
+        it { is_expected.to eq(false) }
+      end
+    end
+
+    describe '#pgde_pgce_with_qts_checked' do
+      subject { described_class.new(params: params).pgde_pgce_with_qts_checked? }
+
+      context 'when PgdePgceWithQts param not present' do
+        let(:params) { { qualifications: %w[Other QtsOnly] } }
+
+        it { is_expected.to eq(false) }
+      end
+
+      context 'when PgdePgceWithQts param is present' do
+        let(:params) { { qualifications: %w[QtsOnly PgdePgceWithQts] } }
+
+        it { is_expected.to eq(true) }
+      end
+
+      context 'when qualifications is empty' do
+        let(:params) { { qualifications: [] } }
+
+        it { is_expected.to eq(false) }
+      end
+
+      context 'when parameters are empty' do
+        let(:params) { {} }
+
+        it { is_expected.to eq(false) }
+      end
+    end
+
+    describe '#other_checked?' do
+      subject { described_class.new(params: params).other_checked? }
+
+      context 'when Other param not present' do
+        let(:params) { { qualifications: %w[QtsOnly PgdePgceWithQts] } }
+
+        it { is_expected.to eq(false) }
+      end
+
+      context 'when Other param is present' do
+        let(:params) { { qualifications: %w[QtsOnly Other] } }
+
+        it { is_expected.to eq(true) }
+      end
+
+      context 'when qualifications is empty' do
+        let(:params) { { qualifications: [] } }
+
+        it { is_expected.to eq(false) }
+      end
+
+      context 'when parameters are empty' do
+        let(:params) { {} }
+
+        it { is_expected.to eq(false) }
+      end
+    end
+
+    describe '#qualification_selected?' do
+      subject { described_class.new(params: params).qualification_selected? }
+
+      context 'when a parameter is selected' do
+        let(:params) { { qualifications: %w[Other] } }
+
+        it { is_expected.to eq(true) }
+      end
+
+      context 'when multiple parameters are selected' do
+        let(:params) { { qualifications: %w[Other QtsOnly] } }
+
+        it { is_expected.to eq(true) }
+      end
+
+      context 'when no parameter is selected' do
+        let(:params) { {} }
+
+        it { is_expected.to eq(false) }
+      end
+    end
+
+    describe '#location_query?' do
+      subject { described_class.new(params: params).location_query? }
+
+      context 'when parameter is present' do
+        let(:params) { { l: '1' } }
+
+        it { is_expected.to eq(true) }
+      end
+
```

Comment:
it might be overkill, but do you think it's worth adding 

 context 'when across England parameter is present' do
        let(:params) { { l: '2' } }

        it { is_expected.to eq(false) }
      end
end 

 context 'when provider parameter is present' do
        let(:params) { { l: '3' } }

        it { is_expected.to eq(false) }
      end
end 

```

Reply:
Yep spec are there on lines 127 and 143

https://github.com/DFE-Digital/find-teacher-training/pull/650/files#diff-0c42db30ea8ea34dff0949b189b5d9e7b9c52eb41499587e1666346624bfc958R127

https://github.com/DFE-Digital/find-teacher-training/pull/650/files#diff-0c42db30ea8ea34dff0949b189b5d9e7b9c52eb41499587e1666346624bfc958R143

Reply:
I think they're testing a different method though right? At the moment we only test that it returns false when it's blank and not that it only returns true if   l == 1. It's really not a big deal anyway though 👍